lib/mina/infinum/ecs/params.rb

```diff
@@ -0,0 +1,66 @@
+require 'mina/infinum/ecs/aws'
+
+namespace :params do
+  desc <<~TXT
+    Print AWS Param Store params to stdout
+
+    Params are fetched from path "/:cluster/*".
+  TXT
+  task read: ['aws:profile:check'] do
+    ensure!(:cluster)
+
+    puts get_params.map(&:as_env)
+  end
+
+  desc <<~TXT
+    Save AWS Param Store params to .env file
+
+    Params are fetched from path "/:cluster/*".
+  TXT
+  task pull: ['aws:profile:check'] do
+    ensure!(:cluster)
+
+    params = get_params
```

Comment:
Do you want the local variable `param` here, or would you prefer to call the `get_params` directly on line 26?

Reply:
Both work for me. I removed the variable 👍